tox.ini

```diff
@@ -145,7 +123,7 @@ passenv =
 deps = {[testenv]deps}
 changedir=test/unit
 commands =
-    bash -c 'cd ../../ && ./setup.py develop'
+    bash -c 'cd ../../ && poetry install'
```

Comment:
No, this is doing things the wrong way. Tox creates a separate virtualenv, in which you call the externally installed poetry, which creates its own virtualenv, bypassing the python version setting of tox and duplicating the work.

Reply:
This is absolutely valid and even documented in Poetry: https://python-poetry.org/docs/faq#use-case-3

Reply:
Poetry will use the tox virtualenv poetry automatically.

Reply:
Poetry will use the virtualenv set up by tox automatically, so it's fine.